packages/ibm-products-styles/src/components/Coachmark/_coachmark-dragbar.scss

```
@@ -0,0 +1,27 @@
+//
+// Copyright IBM Corp. 2023, 2023
```

I will only mark all these _not_ because you need me to but in case it’s easier to batch commit from the suggestions.

```suggestion
// Copyright IBM Corp. 2023, 2024
```